scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,121 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and by default fillet definitions to derive a footprint for gullwing stile packages.
```

Comment:
What does 'by default' add here? Can it be removed? The capability to override IPC7351B fillet parameters would be best in a following sentence. Also, should this mention that the pad style defaults to rounded rectangle, which comes from preliminary IPC7251C documents.

Reply:
changed